packages/frontend/src/components/LightdashVisualisation/index.tsx

```diff
@@ -0,0 +1,38 @@
+import { DBChartTypes } from 'common';
+import EChartsReact from 'echarts-for-react';
+import React, { FC, RefObject } from 'react';
+import { ChartConfig } from '../../hooks/useChartConfig';
+import SimpleChart from '../SimpleChart';
+import SimpleTable from '../SimpleTable';
+
+interface Props {
+    isLoading: boolean;
+    tableName: string | undefined;
+    chartRef: RefObject<EChartsReact>;
+    chartType: DBChartTypes;
+    chartConfig: ChartConfig;
+}
+
+const LightdashVisualisation: FC<Props> = ({
+    isLoading,
+    tableName,
+    chartRef,
+    chartType,
+    chartConfig,
+}) => (
+    <>
+        {chartType === 'table' ? (
+            <SimpleTable />
```

Comment:
Here I would expect `<SimpleTable />` to accept the table data from `chartConfig.plotData` rather than using hooks.

In the explorer view the table gets it's properties from the `useExplore` hook but in a visualisation, the data should be passed straight in.

I think your idea was great to introduce a `SimpleTable` that simply renders some rows of data, without any interaction. 

Then we can have an `ExploreTable` which uses the `useExplore` hook (and many others) to provide that interactive table in the explore view.